Sadi file

saadfareed opened the pull request:
# Application to the Class of 2022🎓

This pull request template helps you complete an application to the **Class of 2022**. Use the checklist below to verify you have followed the instructions correctly. 

## Checklist ✅

- [x] I have read the instructions on the README file before submitting my application. 
- [x] I made my submission by creating a folder on the `_data` folder and followed the naming convention mentioned in the instructions (`<username>`) and markdown file.
- [x] I have submitted a [swag shipping form](https://airtable.com/shrVMo8ItH4wjsO9f).
- [x] I have used the Markdown file template to add my information to the Year Book.
- [x] I understand that a reviewer will merge my pull request after examining it or ask for changes in case needed.
- [x] I understand I should not tag or add a reviewer to this Pull Request.
- [x] I have [added the event](https://calendar.google.com/calendar/u/0/r/eventedit?dates=20220611T200000Z/20220611T220000Z&details&location=https://www.twitch.tv/githubeducation&text=%F0%9F%8E%89%F0%9F%8E%8A+GitHub+Graduation+2022+%F0%9F%8E%89%F0%9F%8E%8A) to my Calendar.

github-education reviewed (changes requested):
**Hi saadfareed,**
**Welcome to graduation!**


### I have a few items I need you to take care of before I can merge this PR:

* *It looks like you still need to fill out the [shipping form](https://airtable.com/shrVMo8ItH4wjsO9f) to continue*
* *Uh Oh! I've found some issues with where you have created your files!* 
	*The required markdown file does not exist, please ensure the file `_data/saadfareed/saadfareed.md` exists*
*Please ensure all changes are contained within the `_data/saadfareed/` directory. Invalid file paths:* 

	* `_data/Sadi/Sadi.md`

* *You need to agree to our COC on the shipping form pretty please!*

Feel free to re-request a review from me and I'll come back and take a look!

saadfareed commented:
Done

saadfareed commented:
check

themagicalmammal committed:
Update themagicalmammal.md

themagicalmammal committed:
Revert "Update themagicalmammal.md"

This reverts commit 306e878088a664e4a00c4e104e0a1f37818bc747.

themagicalmammal committed:
Revert "Revert "Update themagicalmammal.md""

This reverts commit af06ee97c268ed1530849f8d765b85a570642db3.

themagicalmammal committed:
Revert "Revert "Revert "Update themagicalmammal.md"""

This reverts commit 136f00e0088a9d8dafed107955ce625caa9ca863.

themagicalmammal committed:
Revert "Revert "Revert "Revert "Update themagicalmammal.md""""

This reverts commit 41b5811accd0243765c48b4fecd07c0f8e034894.

themagicalmammal committed:
Revert "Revert "Revert "Revert "Revert "Update themagicalmammal.md"""""

This reverts commit a93ab35d426d82036fd564e045304bc3f3fd61f7.

themagicalmammal committed:
Revert "Revert "Revert "Revert "Revert "Revert "Update themagicalmammal.md""""""

This reverts commit 6a4b34189bccb35a26b75fb5c9fec92f306c8234.

themagicalmammal committed:
Revert "Revert "Revert "Revert "Revert "Revert "Revert "Update themagicalmammal.md"""""""

This reverts commit b63e775218f7fa3c273702fd5a636e95979a2ef7.

themagicalmammal committed:
Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Update themagicalmammal.md""""""""

This reverts commit 6057ce8ffa591e2e16c05a72a14081334fd143e7.

themagicalmammal committed:
Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Update themagicalmammal.md"""""""""

This reverts commit 6e017bc2c80e46fc5818414df44bb89310e117bc.

themagicalmammal committed:
Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Update themagicalmammal.md""""""""""

This reverts commit 42cd1213a2772133ce7fb752aada0ec19526e44e.

themagicalmammal committed:
Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Update themagicalmammal.md"""""""""""

This reverts commit 9c88e9c48d97e43dbb95d7439913f8e185ba0dd6.

themagicalmammal committed:
Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Update themagicalmammal.md""""""""""""

This reverts commit f5ac90e644c0bb64d47c59c046d8c71ff0b9c60b.

themagicalmammal committed:
Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Update themagicalmammal.md"""""""""""""

This reverts commit 3da3079c09967f51c526ba5bd5b8225a1210741f.

themagicalmammal committed:
Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Update themagicalmammal.md""""""""""""""

This reverts commit 1d7b8c37ace67ef84be86e36ca12ff3b1d08619f.

themagicalmammal committed:
Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Update themagicalmammal.md"""""""""""""""

This reverts commit 8c22ee73669b59203665ce413a3217f69775f030.

themagicalmammal committed:
Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Update themagicalmammal.md""""""""""""""""

This reverts commit d9b0643584dae0c559c1b3d3ebe998d1cdca6504.

themagicalmammal committed:
Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Update themagicalmammal.md"""""""""""""""""

This reverts commit 0afaa5a82d136c964ce1df0b7e8f46946023a30a.

themagicalmammal committed:
Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Update themagicalmammal.md""""""""""""""""""

This reverts commit c860be05b60e752fc0cebe768740f435ae2ea6a8.

themagicalmammal committed:
Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Update themagicalmammal.md"""""""""""""""""""

This reverts commit 035bd4f4ff58029f5604977339fb09b39897225e.

themagicalmammal committed:
Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Update themagicalmammal.md""""""""""""""""""""

This reverts commit da6d03bd2a9faa5e5772b9731fc87d396a3e6592.

themagicalmammal committed:
Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Update themagicalmammal.md"""""""""""""""""""""

This reverts commit fec77c1e3f77524e558ce527bc97beffeccde66d.

themagicalmammal committed:
Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Update themagicalmammal.md""""""""""""""""""""""

This reverts commit 4b70014f84c214a0250ef6d94bdf5dc6333fb78a.

themagicalmammal committed:
Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Update themagicalmammal.md"""""""""""""""""""""""

This reverts commit b27babd1e316a300eaeec3c82e39bcefac8a68e6.

themagicalmammal committed:
Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Update themagicalmammal.md""""""""""""""""""""""""

This reverts commit 796ea12738a887ef12fa0bcd9f7e7ed540d1d7f1.

themagicalmammal committed:
Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Update themagicalmammal.md"""""""""""""""""""""""""

This reverts commit da36d25342fde11d124c8500c6020357ea6762e2.

themagicalmammal committed:
Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Update themagicalmammal.md""""""""""""""""""""""""""

This reverts commit bbfb566577aaced13655f5da67f70b658ea1ce05.

themagicalmammal committed:
Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Update themagicalmammal.md"""""""""""""""""""""""""""

This reverts commit b617cf3ca2dbf4bb51581e3242d942606c070a26.

themagicalmammal committed:
Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Update themagicalmammal.md""""""""""""""""""""""""""""

This reverts commit 135dcf38c59bb7e1bb1d335160405482039845b5.

themagicalmammal committed:
Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Update themagicalmammal.md"""""""""""""""""""""""""""""

This reverts commit 0655971b42686c26b1565f5c1629ebf994e7febe.

themagicalmammal committed:
Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Update themagicalmammal.md""""""""""""""""""""""""""""""

This reverts commit 5cd8545ec8382939c0e974ef13ea3612d40f19d9.

themagicalmammal committed:
Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Update themagicalmammal.md"""""""""""""""""""""""""""""""

This reverts commit bac21f21c07d3992afd2658c778c255bea398345.

themagicalmammal committed:
Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Update themagicalmammal.md""""""""""""""""""""""""""""""""

This reverts commit 5f77366303e40cae5a932b68ed5f0052fec4adcc.

themagicalmammal committed:
Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Update themagicalmammal.md"""""""""""""""""""""""""""""""""

This reverts commit 1294d224cca3bbf9174c60dca68c8bd8de7c42bc.

themagicalmammal committed:
Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Update themagicalmammal.md""""""""""""""""""""""""""""""""""

This reverts commit 90fdd3ac81468c5d0969cc4cb8cb21252c98cdc8.

themagicalmammal committed:
Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Update themagicalmammal.md"""""""""""""""""""""""""""""""""""

This reverts commit 5e60f7805b213730148b0b546104b51f35c5fce5.

themagicalmammal committed:
Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Update themagicalmammal.md""""""""""""""""""""""""""""""""""""

This reverts commit 6a68938b7ed3feb9e5f312468858ffdd0c6aea2c.

themagicalmammal committed:
Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Update themagicalmammal.md"""""""""""""""""""""""""""""""""""""

This reverts commit fbf573d2075e8f726aeb38d5f9b48e4149b00f3c.

themagicalmammal committed:
Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Update themagicalmammal.md""""""""""""""""""""""""""""""""""""""

This reverts commit ac9b70a93a698aeaa8cec79cec1c0a5276a1907c.

themagicalmammal committed:
Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Update themagicalmammal.md"""""""""""""""""""""""""""""""""""""""

This reverts commit 44704129a2d81333ffcba1456cdaa3fed1a12c06.

themagicalmammal committed:
Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Update themagicalmammal.md""""""""""""""""""""""""""""""""""""""""

This reverts commit 3306f41661920506129f6c92243ff94a5974ff1f.

themagicalmammal committed:
Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Update themagicalmammal.md"""""""""""""""""""""""""""""""""""""""""

This reverts commit 5d3630f3fb9a7776d3dbb93f462d684003fdf95f.

themagicalmammal committed:
Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Update themagicalmammal.md""""""""""""""""""""""""""""""""""""""""""

This reverts commit f59e698b785984cac019649f68a8618ca749257a.

themagicalmammal committed:
Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Update themagicalmammal.md"""""""""""""""""""""""""""""""""""""""""""

This reverts commit 37500c4c7424a0726cb570e8455bb9aa76cff99a.

themagicalmammal committed:
Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Update themagicalmammal.md""""""""""""""""""""""""""""""""""""""""""""

This reverts commit 78ab6d07fb8fc2aca718dfc6e769f94dd476bd86.

themagicalmammal committed:
Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Update themagicalmammal.md"""""""""""""""""""""""""""""""""""""""""""""

This reverts commit d72faa9f6c6f7f4cc967b117d981ee07e12972a3.

themagicalmammal committed:
Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Update themagicalmammal.md""""""""""""""""""""""""""""""""""""""""""""""

This reverts commit d442f56ad9df1bc3c4270ae2e09c370085283c7f.

themagicalmammal committed:
Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Update themagicalmammal.md"""""""""""""""""""""""""""""""""""""""""""""""

This reverts commit 96a6ebf0b0880fd64274a69dc31c4a8176851d42.

themagicalmammal committed:
Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Update themagicalmammal.md""""""""""""""""""""""""""""""""""""""""""""""""

This reverts commit e3cdfa9df543ad84092511afdd70398c2619d2d6.

themagicalmammal committed:
Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Update themagicalmammal.md"""""""""""""""""""""""""""""""""""""""""""""""""

This reverts commit 663c996d26f12ffdebe05cd97dd97bd455cdf49e.

themagicalmammal committed:
Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Update themagicalmammal.md""""""""""""""""""""""""""""""""""""""""""""""""""

This reverts commit 647ec2e1997d7f8428248fd7aa9b8d66f9827a96.

themagicalmammal committed:
Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Update themagicalmammal.md"""""""""""""""""""""""""""""""""""""""""""""""""""

This reverts commit a1a7bc001a27685328ac636fc18b3f0b5b05af03.

themagicalmammal committed:
Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Update themagicalmammal.md""""""""""""""""""""""""""""""""""""""""""""""""""""

This reverts commit 9689b8af3c46976259d5b6aa97fa7d140be53fe2.

themagicalmammal committed:
Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Update themagicalmammal.md"""""""""""""""""""""""""""""""""""""""""""""""""""""

This reverts commit d1b70e3d464557ebf33b272fc3ef612647a5490d.

themagicalmammal committed:
Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Update themagicalmammal.md""""""""""""""""""""""""""""""""""""""""""""""""""""""

This reverts commit a9704262b906d369b578c5effb946bec915b2bcd.

themagicalmammal committed:
Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Update themagicalmammal.md"""""""""""""""""""""""""""""""""""""""""""""""""""""""

This reverts commit 7b7423c52bd7e513f4ff5160b09fad791fc92742.

themagicalmammal committed:
Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Update themagicalmammal.md""""""""""""""""""""""""""""""""""""""""""""""""""""""""

This reverts commit f963b129c13b668cd4bb6f08fa6624ab13284692.

themagicalmammal committed:
Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Update themagicalmammal.md"""""""""""""""""""""""""""""""""""""""""""""""""""""""""

This reverts commit fa902bb4aea8dd173f0e89ad5b54fefb2d839630.

themagicalmammal committed:
Useless commits

- The commits below are completely useless since everything is same from the 1st commit.

khalilhajrii committed:
Create khalilhajrii.md

prathameshThakur committed:
create prathameshThakur.md

Jimmy5467 committed:
Create Jimmy5467.md

JustFossa committed:
Update JustFossa.md

wanisramdani committed:
Create wanisramdani.md

prashimpy committed:
Create Prashim-hack.md

magliocchetti committed:
Create obrobrio2000.md

AyushKoul00 committed:
Create AyushKoul00.md

shalatan committed:
Create shalatan.md

created shalatan.md

devashish-7 committed:
Create devashish-7.md

NehaArora091999 committed:
Create NehaArora091999.md

abhishekapk committed:
Create abhishekapk.md

mikeygman11 committed:
Create mikeygman11.md

Chaitalykundu committed:
Create Chaitalykundu.md

mijoo308 committed:
Create mijoo308.md

jaehoo1 committed:
Create jaehoo1.md

mohanishdc7 committed:
Create mohanishdc7.md

ddamlapinar committed:
Add ddamlapinar

golu-99 committed:
Create golu-99.md

Vishal-raj-1 committed:
add Vishal-raj-1

golu-99 committed:
Update golu-99.md

abhinavsp0730 committed:
Add files via upload

omrajsharma committed:
added Omraj Sharma

Khushik514 committed:
Khushik514 added

mohanishdc7 committed:
Update mohanishdc7.md

prathambhansali committed:
Create pratham-10.md

AlexDev404 committed:
Create AlexDev404.md

prathambhansali committed:
Updated Changes

re-Tick committed:
docs: added re-Tick profile data

AMr4X committed:
Create AMr4X.md

saadfareed committed:
Merge pull request #1 from saadfareed/imgbot

[ImgBot] Optimize images

colinhartigan committed:
wow

colinhartigan committed:
yes

Bikramjit21 committed:
Create Bikramjit21.md

sanjeev0007 committed:
++

vikkastiwari committed:
Create vikkastiwari.md

BimaleshSeth committed:
added my profile

arslanafzalch committed:
Create arslanafzalch.md

Bhavya-Tripathi committed:
Create Bhavya-Tripathi.md

devchopra999 committed:
Add files via upload

grtcoder committed:
HighYoda signing off!!

grtcoder committed:
done

berkekbgz committed:
Create berkekbgz.md

berkekbgz committed:
fixed typo

qwerty22121998 committed:
Create qwerty22121998.md

ra1nst0rm3d committed:
ra1n pull

lifesaver0129 committed:
info added

github-education committed:
Merge pull request #6079 from JackTPatterson/main

Create JackTPatterson

github-education committed:
Merge pull request #7665 from Vi1234sh12/main

Add Vi1234sh12 to Github Graduation 2022

github-education committed:
Merge pull request #7754 from caasperr/caasperr-patch-1

Add caasperr.md file

github-education committed:
Merge pull request #7977 from FreezyEx/main

Create FreezyEx.md

github-education committed:
Merge pull request #8061 from alpdmrel/main

Create alpdmrel.md

github-education committed:
Merge pull request #8063 from david-d25/main

david-d25

github-education committed:
Merge pull request #8064 from AishwaryaHombalimath/main

AishwaryaHombalimath

github-education committed:
Merge pull request #8067 from sudo-pkyt/main

Create sudo-pkyt.md

github-education committed:
Merge pull request #8068 from Jurniko/main

feat: add Jurniko to GitHubGraduation-2022

github-education committed:
Merge pull request #8070 from ENG-AhmedSameh/main

Create ENG-AhmedSameh.md

github-education committed:
Merge pull request #8073 from omzi/main

added my details

github-education committed:
Merge pull request #8074 from shreyask21/main

Added my entry

github-education committed:
Merge pull request #8075 from emily33901/emily33901

Add emily33901

github-education committed:
Merge pull request #8077 from HasanNaser/main

yep 😎

github-education committed:
Merge pull request #8078 from ahuston-0/main

add profile information

github-education committed:
Merge pull request #8079 from ImRodry/patch-1

Add my entry for the class of 22

github-education committed:
Merge pull request #8081 from ray-rafe/main

Graduation

github-education committed:
Merge pull request #8082 from akashjain0900/main

Create akashjain0900.md

github-education committed:
Merge pull request #8083 from benjaminjob/main

Create benjaminjob.md

github-education committed:
Merge pull request #8084 from ClementPeter/main

ClementPeter.md joined Class of 2022

github-education committed:
Merge pull request #8085 from isa56/main

Feat: Add and Submit isa56's markdown

github-education committed:
Merge pull request #8086 from yoruwitch/main

added markdown file

github-education committed:
Merge pull request #8089 from AbilioNB/main

Create AbilioNB.md

github-education committed:
Merge pull request #8095 from LathikaDevraj/main

Create LathikaDevraj.md

github-education committed:
Merge pull request #8096 from GK-INDIA/main

Create GK-INDIA.md

github-education committed:
Merge pull request #8097 from normankong/main

Add normankong

github-education committed:
Merge pull request #8098 from PranavGPR/main

Create PranavGPR.md

github-education committed:
Merge pull request #8102 from DipanForks/main

Fix Quote

Usha103 committed:
Create Usha103.md

Commit:
repository close

Commit:
Merge branch 'main' of https://github.com/education/GitHubGraduation-2022
repository close

elB4RTO committed:
Update README.md

elB4RTO committed:
Update README.md

alphabetical ordered

Commit:
Add Celourence md

shubhi2000 committed:
Update shubhi2000.md

ongzzzzzz committed:
Update ongzzzzzz.md

ongzzzzzz committed:
Merge branch 'education:main' into main

lutforrashidsayed committed:
Update opensourcebangladesh.md

Quote Updated

loidnoir committed:
translation: Armenian translation

Commit:
updated actions

yomaokobiah committed:
Merge pull request #1830 from shubhi2000/main

Create shubhi2000.md

yomaokobiah committed:
Merge pull request #5883 from opensourcebangladesh/main

Github Graduation Year 2022

yomaokobiah committed:
Merge pull request #7591 from ongzzzzzz/main

Create ongzzzzzz.md

yomaokobiah committed:
Merge pull request #8123 from mikeygman11/main

Create mikeygman11.md

yomaokobiah committed:
Merge pull request #8093 from Celourence/main

Add Celourence to GitHubGraduation-2022

yomaokobiah committed:
Merge pull request #8106 from khalilhajrii/main

Create khalilhajrii.md

yomaokobiah committed:
Merge pull request #8107 from Jimmy5467/main

Create Jimmy5467.md

yomaokobiah committed:
Merge pull request #8108 from prathameshThakur/prathameshThakur

create prathameshThakur.md

yomaokobiah committed:
Merge pull request #8113 from obrobrio2000/main

Create obrobrio2000.md

yomaokobiah committed:
Merge pull request #8117 from shalatan/main

Created shalatan.md

yomaokobiah committed:
Merge pull request #8119 from devashish-7/main

Create devashish-7.md

yomaokobiah committed:
Merge pull request #8121 from NehaArora091999/main

Create NehaArora091999.md

yomaokobiah committed:
Merge pull request #8124 from Chaitalykundu/main

Create Chaitalykundu.md

yomaokobiah committed:
Merge pull request #8131 from ddamlapinar/main

Add ddamlapinar

yomaokobiah committed:
Merge pull request #8148 from sanjeev0007/main

Create sanjeev0007.md

yomaokobiah committed:
Merge pull request #8164 from Usha103/main

Create Usha103.md

Commit:
updated actions

Commit:
Merge branch 'main' of https://github.com/education/GitHubGraduation-2022

Commit:
updated actions

Commit:
updated actions

yomaokobiah committed:
Merge pull request #3982 from ajayshinde338/patch-4

Create ajayshinde338.md

yomaokobiah committed:
Merge pull request #7205 from htetpyaeaung/main

Add folder and file

Commit:
updated actions

Commit:
Merge branch 'main' of https://github.com/education/GitHubGraduation-2022

yomaokobiah committed:
Merge pull request #8109 from JustFossa/patch-2

Update JustFossa.md

yomaokobiah committed:
Merge pull request #8111 from Prashim-hack/main

Create Prashim-hack.md

yomaokobiah committed:
Merge pull request #8112 from wanisramdani/main

Create wanisramdani.md

yomaokobiah committed:
Merge pull request #8114 from AyushKoul00/main

Create AyushKoul00.md

yomaokobiah committed:
Merge pull request #8122 from abhishekapk/patch-1

Create abhishekapk.md

Commit:
updated actions

Commit:
Merge branch 'main' of https://github.com/education/GitHubGraduation-2022

yomaokobiah committed:
Merge pull request #8130 from mohanishdc7/main

Create mohanishdc7.md

yomaokobiah committed:
Merge pull request #8137 from omrajsharma/omraj-sharma

added Omraj Sharma

yomaokobiah committed:
Merge pull request #8163 from lifesaver0129/main

Add lifesaver0129 info to main

yomaokobiah committed:
Merge pull request #8161 from mijoo308/main

Create mijoo308.md

yomaokobiah committed:
Merge pull request #8160 from ra1nst0rm3d/main

ra1n pull

yomaokobiah committed:
Merge pull request #8159 from qwerty22121998/qwerty22121998

Create qwerty22121998.md

yomaokobiah committed:
Merge pull request #8157 from berkekbgz/main

Create berkekbgz.md

yomaokobiah committed:
Merge pull request #8134 from golu-99/main

Create golu-99.md

yomaokobiah committed:
Merge pull request #8135 from abhinavsp0730/main

Add files via upload

yomaokobiah committed:
Merge pull request #8133 from Vishal-raj-1/Vishal-raj-1

add Vishal-raj-1

yomaokobiah committed:
Merge pull request #8138 from Khushik514/main

Khushik514 added

yomaokobiah committed:
Merge pull request #8142 from AlexDev404/patch-1

Create AlexDev404.md

yomaokobiah committed:
Merge pull request #8143 from pratham-10/main

updated changes

yomaokobiah committed:
Merge pull request #8144 from re-Tick/main

docs: added re-Tick profile data

yomaokobiah committed:
Merge pull request #8145 from AMr4X/main

Create AMr4X.md

yomaokobiah committed:
Merge pull request #8146 from colinhartigan/main

wow

yomaokobiah committed:
Merge pull request #8147 from Bikramjit21/main

Create Bikramjit21.md

yomaokobiah committed:
Merge pull request #8149 from vikkastiwari/patch-1

Create vikkastiwari.md

yomaokobiah committed:
Merge pull request #8150 from Bimalesh-Seth/main

added my profile

yomaokobiah committed:
Merge pull request #8153 from arslanafzalch/arslanafzalch

Create arslanafzalch.md

yomaokobiah committed:
Merge pull request #8154 from Bhavya-Tripathi/main

Create Bhavya-Tripathi.md

yomaokobiah committed:
Merge pull request #8155 from devansh9999/main

Add files via upload

yomaokobiah committed:
Merge pull request #8156 from grtcoder/main

HighYoda signing off!!

yomaokobiah committed:
Merge pull request #5494 from kenneth-cruz1/kenneth-cruz1-patch-2

Kenneth cruz1 patch 2

yomaokobiah committed:
Merge pull request #8126 from jaehoo1/main

Create jaehoo1.md

mlarasusan committed:
Merge pull request #8062 from gkrockz/main

New PR

mlarasusan committed:
Merge pull request #7279 from Tiger-0512/main

add: Tiger-0512

Commit:
Merge branch 'main' into main

Commit:
Merge pull request #8072 from elB4RTO/main

Create README.it.md

Commit:
Merge branch 'main' into improve-readme-fr

Commit:
Merge pull request #6067 from thomasbnt/improve-readme-fr

Improve some texts on French README

Commit:
Merge pull request #6325 from beatrizrezener/readme_space_fix

Update README.pt-br.md

Commit:
Merge branch 'main' into main

Commit:
Merge pull request #8181 from loidnoir/main

translation: Armenian translation

Commit:
Merge pull request #6194 from gangapurambhargav/main

Create gangapurambhargav.md

Commit:
Merge pull request #6052 from callumok2004/main

Add Callum OKane

Commit:
Merge branch 'main' into patch-1

Commit:
Merge pull request #4318 from ZweiRm/patch-1

Update README.zh.md

Commit:
Update README.md

Commit:
Merge pull request #4406 from phwt/main

Thai translation improvement

Commit:
Merge pull request #5246 from FnrDev/main

fix arabic translation typo

Commit:
Merge pull request #1873 from aryangulati/main

Added aryangulati

loidnoir committed:
translation: Armenian language improvements

thomasbnt committed:
Added authors of FR translation linked to #6067

github-education committed:
Merge pull request #2212 from jayprakash07/main

Submission

github-education committed:
Merge pull request #2794 from ankushKun/main

trying again because of previously graduated issue

github-education committed:
Merge pull request #4283 from qwertyreboot/main

Create qwertyreboot.md

github-education committed:
Merge pull request #4312 from abhinandanarya06/main

Create abhinandanarya06.md

github-education committed:
Merge pull request #4344 from StarTrooper08/add-StarTrooper08

Create StarTrooper.md

github-education committed:
Merge pull request #4486 from Sanan4li/main

added Sanan4li

github-education committed:
Merge pull request #4607 from moli-debugger/main

Create moli-debugger.md

github-education committed:
Merge pull request #4651 from preetvadaliya/patch-1

Added my file.

github-education committed:
Merge pull request #4935 from Naveenkumar0405/main

Create Naveenkumar0405.md

github-education committed:
Merge pull request #4947 from SupaliOjha/main

Create SupaliOjha.md

github-education committed:
Merge pull request #5113 from EmmahCodes254/new_branch

Adding EmmaCodes254 md file

github-education committed:
Merge pull request #5530 from mihirmk/main

Create mihirmk.md

github-education committed:
Merge pull request #5939 from daminebenq/daminebenq-github-graduation-2022

Create daminebenq.md

github-education committed:
Merge pull request #5992 from samuelramox/main

Add samuelramox

github-education committed:
Merge pull request #6021 from Rohan-cod/main

Create Rohan-cod.md

github-education committed:
Merge pull request #6178 from paovazq/main

Added paovazq to the yearbook!

github-education committed:
Merge pull request #6713 from okcd00/main

Join GithubGraduation 2022.

github-education committed:
Merge pull request #6779 from nikhiltripathi1/main

Create nikhiltripathi1.md

github-education committed:
Merge pull request #6964 from harshithkumar2/main

added  harshithkumar2.md

github-education committed:
Merge pull request #7134 from LakinduK/main

Create LakinduK.md

github-education committed:
Merge pull request #7370 from Utsav4852/main

Create Utsav4852.md

github-education committed:
Merge pull request #7668 from arnoob16/main

Created my data folder

github-education committed:
Merge pull request #7857 from dev-rb/main

Graduation Profile.

github-education committed:
Merge pull request #4106 from PradyumnaKrishna/main

Create PradyumnaKrishna.md

github-education committed:
Merge pull request #5361 from supersharmapunit/main

Create supersharmapunit.md

Commit:
Merge pull request #8215 from thomasbnt/add-authors-linked-to-pull-6067

Added authors of FR translation linked to #6067

loidnoir committed:
Information submission update

Commit:
Merge pull request #8206 from loidnoir/patch-1

translation: Armenian language improvements

Commit:
Merge https://github.com/saadfareed/GitHubGraduation-2022
